webrtc-stats.html

```diff
@@ -3844,8 +3110,6 @@ <h3>
              DOMString                protocol;
              required RTCIceCandidateType candidateType;
              long                     priority;
-             DOMString                url;
-             DOMString                relayProtocol;
```

Comment:
both url and relayProtocol are implemented

Reply:
Needs better tests...

Reply:
we should restore these.

Reply:
Restored